src/core_write.cpp

```diff
@@ -227,7 +227,9 @@ void TxToUniv(const CTransaction& tx, const uint256& hashBlock, UniValue& entry,
 
         UniValue out(UniValue::VOBJ);
 
-        out.pushKV("value", ValueFromAmount(txout.nValue));
+        ColorIdentifier colorId(std::move(GetColorIdFromScript(txout.scriptPubKey)));
+        out.pushKV("token", colorId.toHexString());
```

Comment:
The description of the newly added `token` field should also be added to the help for each RPC.